elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -272,9 +281,28 @@
          */
         onModelValueChange: function(newVal){
             this._debug( "urth-core-dataframe onModelValueChange", newVal );
+            newVal = this._deserializeColumnTypes(newVal);
```

Comment:
Since we are checking that newVal exist in the line below... I think we need to protect this line as well